FAQ.md

```diff
@@ -103,6 +103,26 @@ TiDB 目前暂时不支持 select into outfile，可以通过以下方式导出
 
 TiDB 暂不支持数据库层面的会话超时，目前想要实现超时，在没 LB（Load Balancing） 的时候，需要应用侧记录发起的 session 的 id，通过应用自定义超时，超时以后需要到发起 query 的节点上用 kill tidb id 来杀掉 sql。目前建议使用应用程序来实现会话超时，当达到超时时间，应用层就会抛出异常继续执行后续的程序段。
 
+#### TiDB 生产环境的版本管理策略是怎么样的？如何尽可能避免频繁升级？
+TiDB 版本目前逐步标准化，每次 Release 都包含详细的 Change list，版本功能[变化详情](https://github.com/pingcap/TiDB/releases)，生产环境是否有必要升级取决于业务系统，建议升级之前详细了解前后版本的功能差异。
+
+版本号说明参考：`Release Version: v1.0.3-1-ga80e796`，`v1.0.3` 表示 GA 标准版 `-1` 表示该版本 commit 1 次，`ga80e796` 代表版本的 `git-hash`。
+
+#### 分不清 TiDB master 版本之间的区别，经常用错 TiDB-ansible 版本?
+TiDB 目前社区非常活跃，在 GA 版本发布后，还在不断的优化和修改 BUG。因此  TiDB 的版本更新周期比较快，会不定期有新版本发布，请关注我们的[新版本发布官方网站](https://pingcap.com/weekly/ )。此外 TiDB 安装推荐使用 TiDB-Ansible 进行安装，TiDB-Ansible 的版本也会随着 TiDB 的版本发布进行更新，因此建议用户在安装升级新版本的时候使用最新的 TiDB-Ansible 安装包版本进行安装。
+此外，在 TiDB 版本 GA 后，对 TiDB 的版本号进行了统一管理，TiDB 的版本可以通过几种方式进行查看：
++ 通过 `select tidb_version()` 进行查看；
++ 通过执行 `tidb-server -V` 进行查看。
+
+#### 官方有没有三中心跨机房部署的推荐方案？
+从 TiDB 架构来讲，完全支持真正意义上的跨中心异地多活，操作层面依赖数据中心之间的网络延迟和稳定性，一般建议延迟在 5ms 以下，目前我们已经有相似客户方案，具体请咨询官方。
```

Comment:
The "请咨询官方" here refers to a specific team or what else? I think it'd be better to make it more specific so that the users know how to do next. Perhaps an email or phone number ...